Makefile

```diff
@@ -227,3 +228,28 @@ bundle-build:
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
 
 tools: ginkgo kustomize controller-gen operator-sdk
+
+
+api-docs: gen-crd-api-reference-docs kustomize
+	@{ \
+	set -e ;\
+	TMP_DIR=$$(mktemp -d) ; \
+	$(KUSTOMIZE) build config/crd -o $$TMP_DIR/crd-output.yaml ;\
+	$(API_REF_GEN) crdoc --resources $$TMP_DIR/crd-output.yaml --output docs/api.md ;\
+	}
+
+# Find or download gen-crd-api-reference-docs
+gen-crd-api-reference-docs:
+ifeq (, $(shell which crdoc))
+	@{ \
+	set -e ;\
+	API_REF_GEN_TMP_DIR=$$(mktemp -d) ;\
+	cd $$API_REF_GEN_TMP_DIR ;\
+	go mod init tmp ;\
+	go get fybrik.io/crdoc@latest ;\
```

Comment:
Consider pinning to a specific version rather than latest to avoid getting affected by breaking changes.
Also note that gen-crd-api-reference-docs is a different tool so the comment on this target is confusing.
(crdoc maintainer)

Reply:
Thanks for your comments. I used the same as [in this PR](https://github.com/grafana-operator/grafana-operator/pull/521/files) (suggested as a good example to follow in #325). What do you think about the new comment?